policies/coding-style.md

```diff
@@ -749,7 +750,7 @@ by Brian W. Kernighan and Rob Pike.
 Addison-Wesley, Inc., 1999.
 ISBN 0-201-61586-X.
 
-[GNU manuals][5] - where in compliance with K&R and this text - for cpp, gcc,
+[GNU manuals][5] - we're in compliance with K&R and this text - for cpp, gcc,
```

Comment:
"where" didn't seem right, this makes more sense?

Reply:
I'm not even sure what this comment really means